pyvrp/Population.py

```diff
@@ -351,6 +351,17 @@ def select(self) -> Tuple[Individual, Individual]:
 
         return first, second
 
+    def restart(self):
```

Comment:
We could use restart() also in `__init__` to avoid duplicate code but since we're seperating initializing the population anyway in #88 I propose to address it there (i.e. have both `__init__` and `restart` use `self.initialize()`.